Framework/PythonInterface/test/python/plugins/algorithms/HB2AReduceTest.py

```diff
@@ -71,14 +79,28 @@ def test_ExcludeDetectors(self):
     def test_anode_vs_temp(self):
         HB2AReduce_ws = HB2AReduce('HB2A_exp0660_scan0146.dat',
                                    Vanadium='HB2A_exp0644_scan0018.dat',
-                                   IndividualDetectors=True)
+                                   IndividualDetectors=True,
+                                   SaveData=False)
         self.assertTrue(HB2AReduce_ws)
         self.assertEqual(HB2AReduce_ws.getNumberHistograms(), 44)
         self.assertEqual(HB2AReduce_ws.blocksize(), 56)
         self.assertEqual(np.argmax(HB2AReduce_ws.readY(7)), 2)
         self.assertAlmostEquals(np.max(HB2AReduce_ws.readY(7)), 2.13258433)
         HB2AReduce_ws.delete()
 
+    def test_saving_files(self):
+        HB2AReduce_ws = HB2AReduce(
+            'HB2A_exp0660_scan0146.dat',
```

Comment:
```suggestion
            'HB2A_exp0666_scan0024.dat',
```

The file `HB2A_exp0660_scan0146.dat` has  `# def_x = sample` so it will fail (and correctly so) outputting to GSAS format. `HB2A_exp0666_scan0024.dat` should work.

Reply:
I switched to "XYE" format to get pass this error in the sibling PR to ornl-next.  I will mirror the changes to this PR now.